en/subscription_services_privacy_notice.md

```diff
@@ -1,56 +1,65 @@
 # Mozilla Subscription Services | Privacy Notice
 
-Version 1.0, Effective October 11, 2022
-{: datetime="2022-10-11" }
+Version 1.1, Effective November 1, 2023
+{: datetime="2023-11-1" }
```

Comment:
`{: datetime="2023-11-1" }` should be `{: datetime="2023-11-01" }`

Reply:
fixed.